mmv1/products/logging/LogView.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: LogView
+base_url: '{{parent}}/locations/{{location}}/buckets/{{bucket}}/views'
+create_url: '{{parent}}/locations/{{location}}/buckets/{{bucket}}/views?viewId={{name}}'
+self_link: '{{parent}}/locations/{{location}}/buckets/{{bucket}}/views/{{name}}'
+update_verb: :PATCH
+update_mask: true
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    "Official Documentation": "https://cloud.google.com/logging/docs/apis"
+  api: "https://cloud.google.com/logging/docs/reference/v2/rest/v2/projects.locations.buckets.views"
+description: "Describes a view over log entries in a bucket."
+parameters:
+  - !ruby/object:Api::Type::String
+    name: parent
+    description: The parent of the resource.
+    url_param_only: true
+    immutable: true
+  - !ruby/object:Api::Type::String
+    name: location
+    description: 'The location of the resource. The supported locations are: global, us-central1, us-east1, us-west1, asia-east1, europe-west1.'
+    url_param_only: true
+    immutable: true
+  - !ruby/object:Api::Type::String
+    # Make this a String for now since we don't have a good way to reference multiple resources.
+    name: bucket
```

Comment:
diff shows it's losing a diffsuppressfunc `compareSelfLinkOrResourceName`. Relevant?

Reply:
Yes, I've added the diff suppress.